magento/magento2
# Fixed issue #34779: Allow expanding of subcategories on mobile.

### Description (*)
Mobile sub categories are not touchable. After the fix you can expand/collapse sub menus (categories).

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#34779

### Manual testing scenarios (*)

1. Open website via phone or in browser in mobile view (with active touch)
2. Click on "Women" category for example to expand/collapse the category.
3. Expected behavior - see child categories.

### Questions or comments


### Contribution checklist (*)
